sirepo/package_data/nginx_proxy.conf.jinja

```diff
@@ -14,6 +14,8 @@ http {
     client_body_temp_path {{ run_dir }}/client_body_temp;
     proxy_temp_path {{ run_dir }}/proxy_temp;
     fastcgi_temp_path {{ run_dir }}/fastcgi_temp;
+    # uwsgi_temp_path needed for running jupyter
```

Comment:
I think this comment could be expanded to capture more of what's going on. We have to define all of these paths because we are running nginx as a non-priveleged user. nginx default paths expect write level privs on files only writable by root. So, we have to define our own paths in a place writable by us (vagrant). 

I don't think order matters in this part of the file so you can move all paths to the top of the http block and write a comment that covers all of them.

Reply:
fc537f91744023171f36839b42a1c335bed539b9